calc/async_decoupler.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+// Copyright (c) 2016-2017 Tigera, Inc. All rights reserved.
```

Comment:
Is it golang-conventional that we now have packages - calc, proto etc. - directly at the top level of the Git repo?  (Or should there be a top-level dir named 'src', or something similar?)